contracts/ToucanCarbonOffsets.sol

```diff
@@ -66,7 +77,41 @@ contract ToucanCarbonOffsets is
 
     /// @dev Returns the current version of the smart contract
     function version() external pure virtual returns (string memory) {
-        return '1.2.0';
+        return '1.4.0';
+    }
+
+    // ----------------------------------------
+    //      Bridge-related functions
+    // ----------------------------------------
+
+    /// @notice Burn TCO2 on behalf of a user. msg.sender does not require approval
+    /// by the account for the burn to be successfull. This function is exposed so it
+    /// can be utilized in cross-chain transfers of TCO2 where we want to burn the
+    /// TCO2 in the source chain but not retire it.
+    /// @param account The user for whom to burn TCO2
+    /// @param amount The amount to burn.
+    function bridgeBurn(address account, uint256 amount)
```

Comment:
@kargakis are we fine with publishing this? I don't think it's sensitive, as people probably expect TCO2s to be bridged at some point.

Reply:
It's already in a separate branch, see https://github.com/ToucanProtocol/contracts/tree/tco2-bridge-burn - we should have it in master